README.md

```diff
@@ -49,37 +49,53 @@ Lino::CommandLineBuilder.for_command('ls')
     .with_flag('-a')
     .build
     .to_s
-    
+
 # => ls -l -a
-  
+
 # commands with options
 Lino::CommandLineBuilder.for_command('gpg')
     .with_option('--recipient', 'tobyclemson@gmail.com')
     .with_option('--sign', './doc.txt')
     .build
-    .to_s 
-    
+    .to_s
+
 # => gpg --recipient tobyclemson@gmail.com --sign ./doc.txt
-  
-# commands with arguments
+
+# commands with an option repeated multiple times (nil and empty option values are ignored)
```

Comment:
Inclined to remove the bracketed section of the comment (and the nil and '' values in the example)..

Reply:
Agreed, I think the example is self-explanatory